scripts/make_jwk.py

```diff
@@ -78,12 +78,20 @@
                 jwks = {"keys":[]}
 
     # Check if key already present
+    # Check if key already present, before kid check to avoid same key addition
     for key in jwks["keys"]:
```

Comment:
This can now be removed as checking `key==new_key` will always be triggered by the `key["kid"] == new_key["kid"]` check below

Reply:
Do we want to check if the same public key is being added aswell? i.e. `key["n"] == new_key["n"]` just so we don't have the same key duplicated with multiple kid's or does that now matter?

Reply:
I think using the same public key with different `kid`s is OK